tools/system_libs.py

```diff
@@ -932,6 +932,9 @@ def get_cflags(self):
       cflags.append('-D_LIBCXXABI_NO_EXCEPTIONS')
     elif self.eh_mode == Exceptions.EMSCRIPTEN:
       cflags.append('-D__USING_EMSCRIPTEN_EXCEPTIONS__')
+      # The code used to interpret exceptions during terminate
+      # is not compatible with emscripten exceptions.
+      cflags.append('-DLIBCXXABI_SILENT_TERMINATE')
```

Comment:
for code size, should we do the same in native wasm exceptions? or is this useful?

Reply:
This code should work with native exceptions.. and actually I'm hoping we can maybe make it work for emscripten exceptions in a followup.

Reply:
Is it useful, though? That is, how beneficial is the non-silent termination logging?

Reply:
That remains to be seen.  We have at least one user requesting better reporting on uncaught exceptions: https://github.com/emscripten-core/emscripten/issues/6330.

I'm looking at find a way to opt into this (for example in debug builds) without effecting all users/builds.